.gitpod.Dockerfile

```diff
@@ -20,13 +20,6 @@ RUN apt install -y xfce4 xfce4-goodies xserver-xorg-video-dummy xserver-xorg-leg
 # Install python
 RUN apt install -y python3 python3-dev python3-pip python3-setuptools && \
     if [ ! -f "/usr/bin/python" ]; then ln -s /usr/bin/python3 /usr/bin/python; fi
-    
-# Install icub-common
-
-RUN sh -c 'echo "deb http://www.icub.org/ubuntu focal contrib/science" > /etc/apt/sources.list.d/icub.list'
-RUN apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 57A5ACB6110576A6
-RUN apt update
-RUN apt install -y icub-common
```

Comment:
We can now make use of `sudo apt install` on Gitpod; hence, installing dependencies will be part of the assignment on Gitpod.